cmd/soroban-rpc/internal/methods/get_events.go

```diff
@@ -5,25 +5,20 @@ import (
 	"encoding/hex"
 	"encoding/json"
 	"fmt"
-	"io"
-	"net/http"
-	"strconv"
-	"strings"
 	"time"
 
 	"github.com/creachadair/jrpc2"
-	"github.com/creachadair/jrpc2/code"
 	"github.com/creachadair/jrpc2/handler"
-	"github.com/stellar/go/clients/horizonclient"
-	"github.com/stellar/go/protocols/horizon"
+
 	"github.com/stellar/go/support/errors"
-	"github.com/stellar/go/toid"
 	"github.com/stellar/go/xdr"
+
+	"github.com/stellar/soroban-tools/cmd/soroban-rpc/internal/events"
 )
 
 // maxLedgerRange is the maximum allowed value of endLedger-startLedger
 // Just guessed 4320 as it is ~6hrs
-const maxLedgerRange = 4320
+const maxLimit = 10000
```

Comment:
move this const to be configurable value; I'm not particular about the value itself, but would rather all the configuration options to reside in a single place.